### PR TITLE
fix: fix text_type.py exceeds_cap_ratio() returns

### PR DESCRIPTION
**Pull request by jay-ylee**
There are cases when function is_possible_narrative_text receives an incorrect return from function exceeds_cap_ratio and does an incorrect classification, so some of the return values of exceeds_cap_ratio are corrected

### CODE REVIEW

File: unstructured/partition/text_type.py [hunk 1]
```diff
@@ -265,15 +265,17 @@ def exceeds_cap_ratio(text: str, threshold: float = 0.5) -> bool:
     # NOTE(robinson) - Currently limiting this to only sections of text with one sentence.
     # The assumption is that sections with multiple sentences are not titles.
     if sentence_count(text, 3) > 1:
-        logger.debug(f"Text does not contain multiple sentences:\n\n{text}")
+        # logger.debug(f"Text contains multiple sentences:\n\n{text}")
```
Comment (MthwRobinson) [inline unstructured/partition/text_type.py:268]: Can we get rid of this old debug statement?

**Review by MthwRobinson — CHANGES_REQUESTED** [review]
Thanks for this! Could you get add an entry to the changelog under the fixes header and remove the commented out debug statement that I pointed out inline? Will approve and merge once that's in.

File: unstructured/partition/text_type.py (hunk 1 above)
Comment (jay-ylee) [reply unstructured/partition/text_type.py:268]: Of course, gonna change as you say

**Review by MthwRobinson — APPROVED** [review]
LGTM! Thanks for the contribution!

**Pull request merged**